jobs/business-sync/devops/vaults.gcp.env

```diff
@@ -0,0 +1,10 @@
+DATABASE_USERNAME="op://database/$APP_ENV/business-ar-db/DATABASE_USERNAME"
+DATABASE_PASSWORD="op://database/$APP_ENV/business-ar-db/DATABASE_PASSWORD"
+DATABASE_NAME="op://database/$APP_ENV/business-ar-db/DATABASE_NAME"
+DATABASE_UNIX_SOCKET="op://database/$APP_ENV/business-ar-db/DATABASE_UNIX_SOCKET"
+DATABASE_PORT="op://database/$APP_ENV/business-ar-db/DATABASE_PORT"
+WAREHOUSE_DB_UNIX_SOCKET="op://database/$APP_ENV/fin-warehouse/DATABASE_UNIX_SOCKET"
+WAREHOUSE_CREDENTIALS_FILE="op://database/$APP_ENV/fin-warehouse/WAREHOUSE_CREDENTIALS_FILE"
+AUTH_PROXY_CONNECT=
```

Comment:
As I mentioned before these are the vars in 1pass - don't leave undefined vars and make sure you are referencing variables that exist, i.e. DB_READONLY_USER and DB_READONLY_PASSWORD : DATABASE_UNIX_SOCKET="op://database/$APP_ENV/fin-warehouse/DATABASE_UNIX_SOCKET"
DB_READONLY_USER="op://database/$APP_ENV/fin-warehouse/DB_READONLY_USER"
DB_READONLY_PASSWORD="op://database/$APP_ENV/fin-warehouse/DB_READONLY_PASSWORD"